app/src/main/java/org/oppia/android/app/databinding/MarginBindingAdapters.java

```diff
@@ -29,36 +24,18 @@ public static void setLayoutMarginStart(@NonNull View view, float marginStart) {
   public static void setLayoutMarginEnd(@NonNull View view, float marginEnd) {
     if (view.getLayoutParams() instanceof MarginLayoutParams) {
       MarginLayoutParams params = (MarginLayoutParams) view.getLayoutParams();
-      float marginStart = params.getMarginStart();
-      if (isRtlLayout(view)) {
-        setLayoutDirectionalMargins(view, (int) marginEnd, (int) marginStart);
-      } else {
-        setLayoutDirectionalMargins(view, (int) marginStart, (int) marginEnd);
-      }
+      MarginLayoutParamsCompat.setMarginEnd(params, (int) marginEnd);
```

Comment:
FWIW we might not actually need the compat version since it's a min SDK of 17 to directly set end, but I don't mind keeping it.

I actually didn't realize that MarginLayoutParams.setEnd() correctly adjusted the margin based on directionality (its documentation, unfortunately, doesn't indicate this).